lib/classes/class-settings.php

```diff
@@ -113,6 +115,11 @@ public function refresh() {
             $value = constant($constant);
             $this->set( "sm.readonly.{$option}", "constant" );
           }
+          // For key_json, check enviroment variable exists
+          elseif ($option == 'key_json' && $google_app_key_file !== false) {
+            $value = $google_app_key_file;
+            $this->set("sm.readonly.{$option}", "environment");
+          }
```

Comment:
This is wrong. This will just set the value of key_json to file address. We don't need this.